File: Emlak/src/app/admin/model/IKategoriDil.ts
export interface IKategoriDil {
    ID: number,
    //[Required(ErrorMessage = "Baðlý Kategori alaný boþ olamaz.")]
    CatID: number,
    //[Required(ErrorMessage = "Dil alaný boþ olamaz.")]
    TransID: number,
    //[Required(ErrorMessage = "Kategori Adý alaný boþ olamaz ve en fazla 255 karakter olmalýdýr.")]
    //[StringLength(255)]
    CategoryName: string,
    ShortText1: string,
    ShortText2: string,
    //[DataType(DataType.MultilineText)]
    //[AllowHtml]
    Description: string,
    Deleted: boolean,

    Mesaj: string,

    TranslationList: any[],
    CategoryList: any[]
}

File: AdminPanel/Repository/KategoriDil/IKategoriDil.cs
using System.Collections.Generic;
using System.Web.Mvc;

namespace Repository.KategoriDilModel
{
    public interface IKategoriDil
    {
        int ID { get; set; }
        int CatID { get; set; }
        int TransID { get; set; }
        string CategoryName { get; set; }
        string ShortText1 { get; set; }
        string ShortText2 { get; set; }
        string Description { get; set; }
        bool Deleted { get; set; }

        string Mesaj { get; set; }

        List<SelectListItem> TranslationList { get; set; }
        List<SelectListItem> CategoryList { get; set; }
        string CategoryAdi { get; set; }
        string TranslationAdi { get; set; }

        List<KategoriDil> List();
        IKategoriDil Select(int id);
        bool Insert(IKategoriDil kategori);
        bool Update(IKategoriDil kategori);
        bool Delete(int id);
        bool Remove(int id);
    }
}
